<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="es_ES">
<context>
    <name>API</name>
    <message>
        <location filename="../qgiscloudapi/qgiscloudapi.py" line="105"/>
        <source>Failed to access QGIS Cloud API on {api_url}</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>BackgroundLayersMenu</name>
    <message>
        <location filename="../background_layers_menu.py" line="219"/>
        <source>Error</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../background_layers_menu.py" line="219"/>
        <source>A serious error has occurred during the coordinate transformation. Please set the reference system of the project to the WGS84 / Pseudo-Mercartor Projektion (EPSG: 3857) and reload the layer.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>DataUpload</name>
    <message>
        <location filename="../data_upload.py" line="69"/>
        <source>Uploading to database &apos;{db}&apos;...</source>
        <translation>Cargando a la Base de Datos &apos;{db}&apos;...</translation>
    </message>
    <message>
        <location filename="../data_upload.py" line="86"/>
        <source>Database full</source>
        <translation>Base de Datos llena</translation>
    </message>
    <message>
        <location filename="../data_upload.py" line="86"/>
        <source>You have exceeded the maximum database size for your current QGIS Cloud plan. Please free up some space or upgrade your QGIS Cloud plan.</source>
        <translation>Ha excedido el tamaño máximo de la Base de Datos para su plan actual de QGIS Cloud. Libere espacio o actualice su plan de QGIS Cloud.</translation>
    </message>
    <message>
        <location filename="../data_upload.py" line="129"/>
        <source>Creating table &apos;{table}&apos;...</source>
        <translation>Creando Tabla &apos;{table}&apos;...</translation>
    </message>
    <message>
        <location filename="../data_upload.py" line="179"/>
        <source>Uploading features...</source>
        <translation>Cargando Objetos Espaciales...</translation>
    </message>
    <message>
        <location filename="../data_upload.py" line="193"/>
        <source>Feature {id} of layer {layer} has no geometry</source>
        <translation>El Objeto Espacial {id} de la Capa {layer} no contiene Geometría</translation>
    </message>
    <message>
        <location filename="../data_upload.py" line="196"/>
        <source>Feature {id} of layer {layer} has wrong geometry type {type}</source>
        <translation>El Objeto Espacial {id} de la Capa {layer} tiene un tipo de Geometría erroneo {type}</translation>
    </message>
    <message>
        <location filename="../data_upload.py" line="244"/>
        <source>{table}: {count} features uploaded</source>
        <translation>{table}: {count} Objeto Espacial cargado</translation>
    </message>
</context>
<context>
    <name>DbConnectionCfg</name>
    <message>
        <location filename="../db_connection_cfg.py" line="184"/>
        <source>DB Connection Failed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../db_connection_cfg.py" line="184"/>
        <source>Could not connect to the Database &lt;b&gt;%s&lt;/b&gt;&lt;br&gt;&lt;br&gt;. 

Please check the following points:&lt;br&gt;&lt;br&gt;

- Is the computer connected to the Internet?&lt;br&gt;
- Is port 5432 open in the firewall?&lt;br&gt;
- Are the proxy settings correct?</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>ErrorReportDialog</name>
    <message>
        <location filename="../error_report_dialog.py" line="54"/>
        <source>Report error</source>
        <translation>Reporte de Error</translation>
    </message>
</context>
<context>
    <name>LoginDialog</name>
    <message>
        <location filename="../ui_login.ui" line="14"/>
        <source>Login</source>
        <translation>Inicio de Sesión</translation>
    </message>
    <message>
        <location filename="../ui_login.ui" line="20"/>
        <source>User:</source>
        <translation>Usuario:</translation>
    </message>
    <message>
        <location filename="../ui_login.ui" line="43"/>
        <source>Password:</source>
        <translation>Contraseña:</translation>
    </message>
    <message>
        <location filename="../ui_login.ui" line="72"/>
        <source>Token:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_login.ui" line="86"/>
        <source>Copy your token from &lt;a href=&quot;{token_page_url}&quot;&gt;here&lt;/a&gt;</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>MapSettingsDialog</name>
    <message>
        <location filename="../mapsettingsdialog.py" line="45"/>
        <source>Map settings for map: %s</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="228"/>
        <source>Sql Error</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="280"/>
        <source>Add user(s)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="280"/>
        <source>Usernames:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="313"/>
        <source>User error</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="313"/>
        <source>The user(s): %s don&apos;t exist.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="377"/>
        <source>Success</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="321"/>
        <source>The user(s): %s have been added.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="329"/>
        <source>Error</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="329"/>
        <source>No username given.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="359"/>
        <source>Do you really want to delete the user: %s?</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettingsdialog.py" line="377"/>
        <source>The user: %s has been deleted.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>QgisCloudPlugin</name>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="345"/>
        <source>Add background layer</source>
        <translation>Añadir Capa de fondo</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="346"/>
        <source>&lt;i&gt;To add a background layer to the map, install the OpenLayers plugin.&lt;/i&gt;</source>
        <translation>&lt;i&gt;Para agregar una Capa de fondo al mapa, instale el complemento de OpenLayers.&lt;/i&gt;</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="347"/>
        <source>Publish Map</source>
        <translation>Publicar Mapa</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="361"/>
        <source>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Sans Serif&apos;; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;mailto:support@qgiscloud.com?subject=QGISCloud support&quot;&gt;&lt;span style=&quot; font-family:&apos;Ubuntu&apos;; text-decoration: underline; color:#0057ae;&quot;&gt;support@qgiscloud.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="380"/>
        <source>Upload Data</source>
        <translation>Cargando Datos</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="377"/>
        <source>Database:</source>
        <translation>Base de Datos:</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="378"/>
        <source>Refresh layers</source>
        <translation>Actualizar Capas</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="379"/>
        <source>Upload data</source>
        <translation>Cargando Datos</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="398"/>
        <source>Account</source>
        <translation>Cuenta</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="382"/>
        <source>https://api.qgiscloud.com</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="383"/>
        <source>Reset QGIS Cloud API URL</source>
        <translation>Restablecer la URL de API de QGIS Cloud</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="384"/>
        <source>...</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="385"/>
        <source>Login</source>
        <translation>Inicio de Sesión</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="386"/>
        <source>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Ubuntu&apos;; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://qgiscloud.com/account/sign_up&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;Signup&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="391"/>
        <source>Logged in as ...</source>
        <translation>Conectado como ...</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="392"/>
        <source>Logout</source>
        <translation>Cerrar Sesión</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="393"/>
        <source>Databases</source>
        <translation>Bases de Datos</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="394"/>
        <source>Create</source>
        <translation>Crear</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="395"/>
        <source>Delete</source>
        <translation>Eliminar</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="397"/>
        <source>Refresh</source>
        <translation>Actualizar</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="400"/>
        <source>About</source>
        <translation>Acerca de</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="399"/>
        <source>&lt;b&gt;Plugin version:&lt;/b&gt;</source>
        <translation>&lt;b&gt;Versión del Plugin:&lt;/b&gt;</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="376"/>
        <source>Maps</source>
        <translation>Mapa</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="349"/>
        <source>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Ubuntu&apos;; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://wms.qgiscloud.com/user/map&quot;&gt;&lt;span style=&quot; font-size:9pt; text-decoration: underline; color:#0057ae;&quot;&gt;https://wms.qgiscloud.com/user/map&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="355"/>
        <source>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Ubuntu&apos;; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://qgiscloud.com/maps&quot;&gt;&lt;span style=&quot; font-size:9pt; text-decoration: underline; color:#0057ae;&quot;&gt;https://qgiscloud.com/maps&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="367"/>
        <source>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Ubuntu&apos;; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://qgiscloud.com/user/map&quot;&gt;&lt;span style=&quot; font-size:9pt; text-decoration: underline; color:#0057ae;&quot;&gt;https://qgiscloud.com/user/map&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="373"/>
        <source>Published Maps</source>
        <translation>Mapas Publicados</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="374"/>
        <source>Open Project</source>
        <translation>Proyecto Abierto</translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="381"/>
        <source>&amp;Server:</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="344"/>
        <source>QGIS Cloud</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="375"/>
        <source>Map Settings</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="348"/>
        <source>Public WMS/WFS/WCS:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="354"/>
        <source>Map Admin:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="360"/>
        <source>Support:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="366"/>
        <source>Webmap:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_qgiscloudplugin.py" line="396"/>
        <source>DB Tables</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>QgisCloudPluginDialog</name>
    <message>
        <location filename="../db_connection_cfg.py" line="138"/>
        <source>host: %s port: %s database: %s username: %s password: %s</source>
        <translation>host: %s puerto: %s base de datos: %s usuario: %s contraseña: %s</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="401"/>
        <source>New Version</source>
        <translation>Versión Actual</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="401"/>
        <source>New plugin release {version} is available! Please upgrade the QGIS Cloud plugin.</source>
        <translation>¡Ya está disponible la nueva versión {version} del plugin! Actualice el complemento QGIS Cloud.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1045"/>
        <source>QGIS Cloud</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="430"/>
        <source>Unsupported versions detected. Please check your versions first!</source>
        <translation>Se detectaron versiones no soportadas. ¡Primero compruebe sus versiones!</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="438"/>
        <source>Account Disabled</source>
        <translation>Cuenta Deshabilitada</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="438"/>
        <source>Account {username} is disabled! Please contact support@qgiscloud.com</source>
        <translation>¡La cuenta del usuario {username} esta deshabilitada! Favor de contactar a support@qgiscloud.com</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="447"/>
        <source>Login for user {username} failed</source>
        <translation>Falló el inicio de sesión para del usuario {username}</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="451"/>
        <source>Wrong user name or password</source>
        <translation>Nombre de usuario o contraseña incorrecto</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="460"/>
        <source>Login failed</source>
        <translation>Fallo de inicio de sesión</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="455"/>
        <source>Login failed: %s</source>
        <translation>Falló el inicio de sesión: %s</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="983"/>
        <source>Warning!</source>
        <translation>¡Advertencia!</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="514"/>
        <source>Number of %s permitted databases exceeded! Please upgrade your account!</source>
        <translation>¡El número permitido de %s bases de datos ha sido excedido! ¡Actualice su cuenta!</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1002"/>
        <source>Warning</source>
        <translation>Advertencia</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="562"/>
        <source>Delete QGIS Cloud database.</source>
        <translation>Eliminar la Base de Datos QGIS Cloud.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="658"/>
        <source>No databases</source>
        <translation>No hay Bases de Datos</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="668"/>
        <source>Select database</source>
        <translation>Seleccinar Base de Datos</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="850"/>
        <source>Project Modified</source>
        <translation>Proyecto Modificado</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="851"/>
        <source>The project has been modified.</source>
        <translation>El Proyecto ha sido modificado.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="852"/>
        <source>The project needs to be saved before it can be published. Proceed?</source>
        <translation>El proyecto necesita ser guardado antes de que pueda ser publicado. ¿Desea guardar el proyecto?</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="938"/>
        <source>The project has a user defined CRS. The use of user defined CRS is not supported. Please correct the project CRS before publishing!</source>
        <translation>El proyecto tiene un CRS definido por el usuario. No se admite el uso de CRS definido por el usuario. ¡Favor de corregir el CRS del proyecto antes de publicarlo!</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1008"/>
        <source>Cancelled</source>
        <translation>Cancelado</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1010"/>
        <source>Publishing map</source>
        <translation>Publicar Mapa</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1047"/>
        <source>Map successfully published</source>
        <translation>El Mapa ha sido publicado con éxito</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1051"/>
        <source>Error uploading project</source>
        <translation>Error al cargar el proyecto</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1096"/>
        <source>An error occured.</source>
        <translation>Ha ocurrido un error.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1057"/>
        <source>Uploading SVG symbols</source>
        <translation>Cargando simbolos SVG</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1096"/>
        <source>Error checking local data sources</source>
        <translation>Error al comprobar fuentes de datos locales</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1108"/>
        <source>Local layers found</source>
        <translation>Se han encontrado capas locales</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1113"/>
        <source>Unsupported layers found</source>
        <translation>Se han encontrado capas que no son compatibles</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1117"/>
        <source>
Please remove or replace above layers before publishing your map.
</source>
        <translation>Favor de remover o reemplazar las capas anteriores antes de publicar su mapa.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1175"/>
        <source>Unsupported geometry type</source>
        <translation>Tipo de geometría no soportado</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1175"/>
        <source>Unsupported geometry type &apos;{type}&apos; in layer &apos;{layer}&apos;</source>
        <translation>Tipo de geometría no soportado &apos;{type}&apos; en la Capa &apos;{layer}&apos;</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1181"/>
        <source>Note: OGR features will be converted to MULTI-type</source>
        <translation>Nota: Las características de OGR se convertirán en MULTI-type</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1235"/>
        <source>Updated local data sources</source>
        <translation>Fuentes de datos locales actualizadas</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1271"/>
        <source>Unknown type</source>
        <translation>Tipo desconocido</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1391"/>
        <source>No database available</source>
        <translation>No hay Base de Datos disponible</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1391"/>
        <source>Please create a database in the &apos;Account&apos; tab.</source>
        <translation>Favor de crear una Base de Datos desde el menú &apos;Cuenta&apos;.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1396"/>
        <source>No database selected</source>
        <translation>No se ha seleccionado una Base de Datos</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1396"/>
        <source>Please select a database to upload data.</source>
        <translation>Favor de seleccionar una base de datos para cargar datos.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1406"/>
        <source>Network Error</source>
        <translation>Error de Red</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1406"/>
        <source>Could not connect to database server ({0}) on port {1}. Please contact your system administrator or internet provider to open port {1} in the firewall</source>
        <translation>No se pudo realizar la conección al servidor de base de datos ({0}) en el puerto {1}. Póngase en contacto con el administrador del sistema o el proveedor de Internet para abrir el puerto {1} en el firewall</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1413"/>
        <source>Uploading data...</source>
        <translation>Cargando Datos...</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1441"/>
        <source>Upload errors occurred</source>
        <translation>Ha ocurrido un error al cargar los datos</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1441"/>
        <source>Upload errors occurred. Not all data could be uploaded.</source>
        <translation>Errores en la carga de datos. No todos los datos han podido ser cargados.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1478"/>
        <source>Save Project</source>
        <translation>Guardar el Proyecto</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1465"/>
        <source>Upload complete. The local layers in the project were replaced with the layers uploaded to the qgiscloud database.</source>
        <translation>Carga completa. Las capas locales del proyecto se sustituyeron por las capas cargada a la base de datos qgiscloud.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1469"/>
        <source>Choose were to save the modified project:</source>
        <translation>Elija la ubicación donde se guardará el proyecto modificado:</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1478"/>
        <source>QGIS Project Files</source>
        <translation>Archivos del Proyecto QGIS</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1505"/>
        <source>QGIS Cloud Error</source>
        <translation>Error QGIS Cloud</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1507"/>
        <source>Error</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1592"/>
        <source>Used DB Storage: </source>
        <translation>Espacio usado en la Base de Datos:</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1109"/>
        <source>Some layers are using local data. Please upload local layers to your cloud database in the &apos;Upload Data&apos; tab before publishing.

</source>
        <translation>Algunas capas utilizan datos locales. Cargue las capas locales en su base de datos de la nube desde el menú &apos;Cargar Datos&apos; Antes de publicar.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="160"/>
        <source>Privacy Policy</source>
        <translation>Política de privacidad</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="378"/>
        <source>Accept new Privacy Policy</source>
        <translation>Acepto la nueva Política de Privacidad</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="411"/>
        <source>Logged in as {0} ({1})</source>
        <translation>Conectado como {0} ({1})</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="422"/>
        <source>Logged in as {0}</source>
        <translation>Conectado como {0}</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="544"/>
        <source>You have layers from database &quot;{name}&quot; loaded in your project! Do you want to remove them before you delete database &quot;{name}&quot;?</source>
        <translation>Tiene capas de la base de datos &amp;quot;{name}&amp;quot; cargadas en su proyecto! ¿Desea eliminarlos antes de eliminar la base de datos &amp;quot;{name}&amp;quot;?</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="557"/>
        <source>Deletion of database &quot;{name}&quot; interrupted!</source>
        <translation>Borrado de la base de datos &amp;quot;{name}&amp;quot; interrumpido!</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="563"/>
        <source>Do you want to delete the database &quot;%s&quot;?</source>
        <translation>¿Desea borrar la base de datos &amp;quot;%s&amp;quot;?</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="689"/>
        <source>Your actual project has changes. Do you want to save the project?</source>
        <translation>Su proyecto actual tiene cambios. ¿Quieres salvar el proyecto?</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="712"/>
        <source>Delete QGIS Cloud map.</source>
        <translation>Eliminar el mapa de QGIS Cloud.</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="713"/>
        <source>Do you want to delete the map &quot;%s&quot;?</source>
        <translation>¿Desea borrar el mapa &amp;quot;%s&amp;quot;?</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="736"/>
        <source>Deletion of map &quot;{name}&quot; interrupted!</source>
        <translation>Se interrumpe el borrado del mapa &amp;quot;{name}&amp;quot;!</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1115"/>
        <source>  -  %s (%s)
</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="378"/>
        <source>Due to the GDPR qgiscloud.com has a new &lt;a href=&apos;http://qgiscloud.com/en/pages/privacy&apos;&gt; Privacy Policy &lt;/a&gt;.
                                To continue using qgiscloud.com, you must accept the new policy. </source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="934"/>
        <source>The project has an unknown CRS. Please set a valid map CRS in Project-&gt;Properties...-&gt;CRS before publishing!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="973"/>
        <source>The following WMS layer names are not unique: {}. Please make sure the names are unique, then publish the project again. The layer WMS short name can be set in the layer properties dialog under &apos;QGIS Server -&gt; Short name&apos;.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="467"/>
        <source>Could not resolve Host Name</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="467"/>
        <source>Could not access {url}. Please check that the URL is written correctly. The error: was &apos;{e}&apos;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="477"/>
        <source>No Database</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="477"/>
        <source>
To work with QGIS Cloud you need at least one QGIS Cloud database. Creating a database can take a few minutes, please be patient.

Do you want to create a new database now?
</source>
        <translation>
Para trabajar con QGIS Cloud necesita al menos una base de datos de QGIS Cloud. La creación de una base de datos puede tardar unos minutos, por favor sea paciente.

Quieres crear una nueva base de datos ahora?</translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="274"/>
        <source>The map name has been changed to &apos;{name}&apos; to conform to allowed characters ({allowed_chars}).</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="415"/>
        <source>Logged in as {0} ({1})
Paid until: {2}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="445"/>
        <source>Invalid authentication token</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="957"/>
        <source>The following layer names do contain non allowed characters like &apos;, . : ;&apos; : {}. Please rename the layers and save the project before publishing.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="965"/>
        <source>The following layer and/or group names are not unique in the project:

{}

Please rename the layers and/or the groups and save the project before publishing.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="983"/>
        <source>The layer(s) {layerlist} have user defined CRS. The use of user defined CRS is not supported. Please correct the CRS before publishing!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="995"/>
        <source>
The CRS of the background layer &apos;{layerName}&apos; is different to the  map CRS. This means that this layer will be reprojected in the published map and the webmap will therefore be slow. To improve this and make the webmap faster, go to

&apos;Project -&gt; Properties... -&gt; CRS&apos;

and set the map CRS to {layerCRS}. Continue publishing?</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1026"/>
        <source>
The name of the QGIS project

%s

is invalid. It has the extension &apos;qgs.qgz&apos;. This is not allowed. Please correct the project name and publish the project again.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1190"/>
        <source>QGIS Cloud supports only Authority EPSG, IAU_2015 or ESRI reference systems</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1335"/>
        <source>Table name too long</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgiscloudplugindialog.py" line="1335"/>
        <source>The name of table

{table_name}

has more than 62 characters and cannot be processed like this. Please give the table a shorter name.

Should the table name be shortened automatically?
</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>RasterUpload</name>
    <message>
        <location filename="../raster/raster_upload.py" line="100"/>
        <source>Database full</source>
        <translation>Base de Datos llena</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="100"/>
        <source>Upload would exceeded the maximum database size for your current QGIS Cloud plan. Please free up some space or upgrade your QGIS Cloud plan.</source>
        <translation>La carga de datos excedería el tamaño máximo de la base de datos para su plan de QGIS Cloud actual. Libere espacio o actualice su plan de QGIS Cloud.</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="107"/>
        <source>Creating table &apos;{table}&apos;...</source>
        <translation>Creando Tabla &apos;{table}&apos;...</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="131"/>
        <source>Creating overview-level {level} for table &apos;{table}&apos;...</source>
        <translation>Creación de nivel de vista general {level} para la Tabla &apos;{table}&apos;...</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="145"/>
        <source>Registering raster columns of table &apos;%s&apos;...</source>
        <translation>Registro de columnas Ráster de la Tabla &apos;%s&apos;...</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="646"/>
        <source>Uploading tiles...</source>
        <translation>Cargando bloques...</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="708"/>
        <source>Calculating raster params for {sum_tiles} tiles ...</source>
        <translation>Cálculo de parametros raster para {sum_tiles} tiles ....</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="141"/>
        <source>Creating GIST Index for table &apos;{table}&apos;...</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>RelationSizeDialog</name>
    <message>
        <location filename="../ui_relation_size.ui" line="14"/>
        <source>Table sizes</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_relation_size.ui" line="58"/>
        <source>Export CSV</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../ui_relation_size.py" line="79"/>
        <source>Total amount of used tablespace %s</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>map_settings</name>
    <message>
        <location filename="../mapsettings.ui" line="14"/>
        <source>Map Settings</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="20"/>
        <source>General Settings</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="53"/>
        <source>Map public</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="104"/>
        <source>Viewer</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="123"/>
        <source>Scales</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="139"/>
        <source>Language</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="152"/>
        <source>Viewer active</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="165"/>
        <source>WMS public</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="183"/>
        <source>Allowed Users</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="235"/>
        <source>Search Settings</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="246"/>
        <source>SQL for search with return values displaytext and bbox in map coordinates, e.g. </source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="267"/>
        <source>Search type</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="280"/>
        <source>Search DB</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="313"/>
        <source>Search SQL</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="331"/>
        <source>SELECT name AS displaytext, Box2D(st_transform(wkb_geometry, &lt;CRS&gt;)) AS bbox 
FROM searchtable 
WHERE name ILIKE &apos;%?%&apos;;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="343"/>
        <source>Share Map</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="357"/>
        <source>Share with</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../mapsettings.ui" line="376"/>
        <source>CheckBox</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
